PURL-TYPES.rst

```diff
@@ -428,7 +428,19 @@ mlflow
       pkg:mlflow/creditfraud@3?repository_url=https://westus2.api.azureml.ms/mlflow/v1.0/subscriptions/a50f2011-fab8-4164-af23-c62881ef8c95/resourceGroups/TestResourceGroup/providers/Microsoft.MachineLearningServices/workspaces/TestWorkspace
       pkg:mlflow/trafficsigns@10?model_uuid=36233173b22f4c89b451f1228d700d49&run_id=410a3121-2709-4f88-98dd-dba0ef056b0a&repository_url=https://adb-5245952564735461.0.azuredatabricks.net/api/2.0/mlflow
 
+nipkg
+-----
+``nipkg`` for NI Package format packages:
+
+- The ``namespace`` is the maintainer from the package attributes (optional).
```

Comment:
What does it mean for the maintainer to be optional? Is it optional because the maintainer has a default?

If it's optional because it doesn't matter, it shouldn't be included because that creates problems for code that needs to match PURLs.